go/runtime/host/sgx/epid.go

```diff
@@ -52,17 +44,54 @@ func (ep *teeStateEPID) Update(ctx context.Context, sp *sgxProvisioner, conn pro
 	}
 	supportsAttestationV1 := (regParams.TEEFeatures != nil && regParams.TEEFeatures.SGX.PCS)
 
+	// TODO: Could consider implementing a strategy for more optimized endpoint selection,
```

Comment:
Agree, we don't need to do anything advanced, but we could at least start where we ended.

Reply:
Added, check again.